packages/react/src/Footer/FooterMenu.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { LoginIcon } from '@amsterdam/design-system-react-icons'
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { FooterMenu } from './FooterMenu'
+import '@testing-library/jest-dom'
+import { FooterMenuLink } from './FooterMenuLink'
+
+describe('Page menu', () => {
+  it('renders a page menu with children', () => {
+    const { container } = render(
+      <FooterMenu>
```

Comment:
Using Footer.Menu etc. here may be slightly more realistic – we do so in other subcomponents.

Reply:
Changed the other tests too